packages/stark-build/config/webpack.dev.js

```diff
@@ -26,10 +26,17 @@ const WebpackMonitor = require("webpack-monitor");
  *
  * See: http://webpack.github.io/docs/configuration.html#cli
  */
-module.exports = function() {
+module.exports = function(env, argv) {
+	console.log(env);
 	const ENV = (process.env.ENV = process.env.NODE_ENV = "development");
 	const HOST = process.env.HOST || "localhost";
 	const PORT = process.env.PORT || 3000;
+	let MONITOR = false;
+	if (typeof env !== "undefined") {
```

Comment:
2 things:

1.- What is received in the "env" parameter when it is called in the npm script with the `-- --env.monitor`?
2.- This `if` block can be flattened: 

```
if (env && typeof env.monitor !== "undefined") {
  MONITOR = env.monitor;
}
```

Reply:
1.- https://webpack.js.org/api/cli/#environment-options